static/js/components/PersonalTab.js

```diff
@@ -30,7 +35,7 @@ class PersonalTab extends React.Component {
       <Grid className="profile-tab-grid">
         <Cell col={1} />
         <Cell col={10}>
-          <PersonalForm {...this.props} />
+          <PersonalForm {...this.props} validator={personalValidation} />
```

Comment:
any reason that we can't just import `personalValidation` directly in `PersonalForm` and get rid of the import here?